forte/data/data_store.py

```diff
@@ -474,3 +476,85 @@ def prev_entry(self, tid: int) -> List:
             The previous entry of the same type as the ``tid`` entry.
         """
         raise NotImplementedError
+
+    def parse_onto_file(self):
```

Comment:
Try to reuse the ontology parsing code from [OntologyGenerator](https://github.com/asyml/forte/blob/master/forte/data/ontology/ontology_code_generator.py#L191)